compiler/testData/diagnostics/tests/testsWithJava15/jvmRecord/simpleRecords.fir.kt

```diff
@@ -8,7 +8,7 @@ public record MyRecord(int x, CharSequence y) {
 // FILE: main.kt
 
 fun foo(mr: MyRecord) {
-    <!INAPPLICABLE_CANDIDATE!>MyRecord<!>(1, "")
+    <!TOO_MANY_ARGUMENTS!>MyRecord<!>(1, "")
```

Comment:
This shouldn't be `TOO_MANY_ARGUMENTS`: https://github.com/JetBrains/kotlin/blob/master/compiler/testData/diagnostics/tests/testsWithJava15/jvmRecord/simpleRecords.kt#L11

Reply:
Ah, it was `INAPPLICABLE_CANDIDATE` anyway, so the question is why/where the root cause was set as `TooManyArguments`.